### ISSUE TITLE
Not working ..

### ISSUE THREAD

**Issue by Sagem0** [sfDesat/Celestial-Tint#2]
Hello !
Thanks for your work :D 

Does the mod work ONLY for custom moons ?

Because i tried everything and when i'm in orbit everything is black :c
![Screenshot_1](https://github.com/sfDesat/Celestial-Tint/assets/125790889/f648ef04-20bd-48ff-b7d0-64bd3bcd8067)

**Comment by sfDesat** [comment]
Can you send your modlist?

**Comment by Sagem0** [comment]
Hmm.. can you tell me how to do that please ? ^^'

**Comment by Sagem0** [comment]
Nevermind !
I guess i found it 

```
BepInEx-BepInExPack-5.4.2100
Sligili-HDLethalCompany-1.5.6
monkes_mods-JumpDelayPatch-1.0.1
taffyko-NameplateTweaks-1.0.6
BepInEx-BepInEx_MLLoader-2.1.0
Owen3H-IntroTweaks-1.5.0
AllToasters-SpectateEnemies-2.3.1
2018-LC_API-3.4.5
RugbugRedfern-Skinwalkers-4.0.3
ZTK-ZTKCosmetics-2.0.0
Evaisa-HookGenPatcher-0.0.5
SocomRazorback-RandysSuits-1.1.0
no00ob-LCSoundTool-1.5.1
Clementinise-CustomSounds-2.3.2
EliteMasterEric-Coroner-1.6.2
itsmeowdev-DoorFix-1.0.0
x753-More_Suits-1.4.1
Verity-TooManySuits-1.0.5
Hexnet111-SuitSaver-1.1.4
M1GDevin-M1GDevin_Collection_of_suits-1.0.3
Evaisa-LethalLib-0.14.2
quackandcheese-MirrorDecor-1.3.2
Hexa-Hexas_Cosmetics-0.0.2
PopleZoo-BetterItemScan-3.0.2
BreakfastBash-BreakfastCosmetics-1.1.1
MadR-MadnessPN2cosmetics-1.0.0
Rune580-LethalCompany_InputUtils-0.6.3
Ozone-Runtime_Netcode_Patcher-0.2.5
h410pr0-Halo_Alpha_9_Cosmetic_Pack-1.0.1
willis81808-LethalSettings-1.4.0
AinaVT-LethalConfig-1.3.4
FlipMods-TooManyEmotes-2.0.3
GhostbustingTrio-CustomFunSuits-1.8.3
JordinB-RandomSuits-1.0.1
ViViKo-MoreMaterials-1.3.2
ViViKo-ViViKoCosmetics-1.5.0
cahloe-Odd_Cosmetics-1.0.2
broiiler-inacraft_cosmetics_megapack-1.0.1
NiaNation-AbsasCosmetics-1.2.3
boring-EmoteWhileInMotion-1.0.0
Speed3559-LethalCosmetics-1.0.0
HomelessGinger-MaskedEnemyOverhaul-3.1.2
EliteMasterEric-CleanerLogs-1.0.0
aazza94-NefariousCosmeticPack-1.0.2
Spyci-CozyImprovements-1.2.2
Fredolx-SignalTranslatorUpgrade-1.0.0
LeKAKiD-SignalTranslatorAligner-1.1.0
ButteryStancakes-AccurateStaminaDisplay-2.0.1
Woecust-Immersive_Visor-0.2.1
BunyaPineTree-ModelReplacementAPI-2.3.8
N34r-Traduction_FR_de_Lethal_Company-1.3.4
Backrooms-Backrooms-0.1.3
Pooble-LCBetterSaves-1.7.3
teamthing-indies_suits-1.6.0
InkCruxStudios-Ender_MoreSuits-1.0.0
WalTeam-NoCooldownInverseTeleporter-1.0.0
Beepsterr-AnnounceMoonChange-1.0.1
Spantle-ThrowEverything-0.4.1
Zaggy1024-PathfindingLagFix-1.2.1
Nikki-NikkisCosmeticKingdom-2.0.0
TesTeam-V2Cosmetics-1.1.1
Hardy-LCMaxSoundsFix-1.1.0
SpookyFingas-Spookies_Cosmetics-3.0.0
KRILLCREW-BetterMenuMusic-2.0.0
Mellowdy-HandheldMap-1.6.1
Zaggy1024-OpenBodyCams-1.3.0
Aavild-NewTerminal-1.4.0
Sligili-More_Emotes-1.3.3
scoopy-Scoopys_Variety_Mod-1.2.0
Justice69-ImmersiveScraps-1.2.0
bigmcnugget-VanillaContentExpansion-0.1.8
RaspberryProgramming-NoDeathDespawn-1.0.0
Taker-CannabisCompany-1.0.3
Sakura-LandminesForAll-1.0.1
thechitoteam-SpectatorCameraConfig-1.0.2
Aelstraz-BetterSpectator-1.2.0
vasanex-ItemQuickSwitch-1.1.0
Major_And_Skiz-MoreInteriors-2.5.0
taffyko-BetterSprayPaint-2.0.2
Swaggies-CustomDeathMessage-1.0.0
Bobbie-LethalRebinding-1.0.0
Supremo-Hit_Haste-1.0.0
Lauriichan-FriendPatches-1.0.3
Flowprojects-AoTGiants-1.1.0
snake_tech-HazardsOutside-1.1.3
mrov-LightsOut-0.0.6
PortableNavi-BetterInverseTeleporter-1.0.2
NutNutty-AlwaysPickup-1.2.0
Mhz-DisableUnityLogs-1.0.0
falcodxb-LessLogs-1.0.1
kz-RemoveMotionSway-1.2.3
an0nymooose-An0n_Patches-1.0.4
Zehs-Hitmarker-1.1.5
QuokkaCrew-MoreMonsters-1.2.1
TanmanG-Forest_Giant_Motionsense-1.0.2
Sagemo-audiopackforfriends8-1.0.8
IntegrityChaos-Diversity-1.1.10
lukeprime-CrouchHUD-1.0.0
zealsprince-Locker-1.1.0
KirbyMadness-Kirby_Cosmetics-1.5.0
DeathWrench-PingDisplay-1.3.0
veri-ShipWindows-2.0.0
Rattenbonkers-TVLoader-1.1.1
mrov-LethalRichPresence-0.5.8
DOLTISH-DragonBallClips-1.1.0
sfDesat-Celestial_Tint-1.1.3
Wolf11221-SpaceShipDoor-1.0.1
IAmBatby-LethalLevelLoader-1.1.6
jockie-LethalExpansionCore-1.3.15
```

**Comment by sfDesat** [comment]
I recommend checking ShipWindows config and make sure these are set like this.  
![image](https://github.com/sfDesat/Celestial-Tint/assets/75356901/9749e30a-e12a-48b6-bcb2-93a9f6cb8b7d)
If that doesn't work, maybe try disabling LightsOut? That may be messing with the outside.

**Comment by Sagem0** [comment]
I just did that, and it doesnt work either ... 
I dont know what cause the issue :/

Edit :
Even if i remove CelestialTint and set SpaceOutside to 1 or 2 it changes nothing 

Edit2 :
So i tested with a clean install, only with your mod and the dependencies and it worked.
I'll test on my side and find what causes the issue :)

Edit3 :
So i did reinstall all the mods that can be see in the modlist above 10 per 10
Everything worked fine.
BUT, i did take all of previous config files and paste them info the new modpack -> and CelestialTint did not work anymore.
I'm investigating :]

Edit4 :
I did not find the culprit
BUT
In my opinion it is related to other interior mods
as Major_And_Skiz-MoreInteriors-2.5.0
or scoopy-Scoopys_Variety_Mod-1.2.0

**Comment by sfDesat** [comment]
thank you for doing some investigation. I will also do some investigating

**Comment by Katsugami** [comment]
Hello,

I'm reaching out again because I'm facing the same issue. Thus, I conducted a search and deactivated all mods, initially launching only what's necessary for Celestia mode. 
Great, it worked.
Then, I activated them in batches of 5 until it stopped working.
Afterwards, I proceeded to activate them one by one.
The first mod I found that caused a crash is "x753-More_Suits-1.4.1".

I'm continuing without the More_suits mod to see if there are any others.

Edit 1 : I have just finished, and it's indeed "x753-More_Suits-1.4.1" that no longer works with "Celestia". 

Edit 2 : After several modifications, I haven't been able to get "more_suits" to work with "celestia" (and all the other mods enabled). So, I conducted a test: I disabled all the mods and reactivated just "celestia" + "more_suit", and it worked. Therefore, I am now searching for what could be causing "more_suits" to bug, which is causing a problem for "celestia".

Edit 3 : I just found out: NewTerminal causes more_suits to bug, and more_suits causes celestia to bug.
